abstracts/abstract_en.tex

```diff
@@ -8,7 +8,16 @@
     It has been employed extensively in acclaimed titles such as \textit{Minecraft} and \textit{No Man's Sky} to create virtually infinite worlds that the player is free to interact with.
     On the other hand, a recent game \textit{Hyperbolica} popularized a novel idea of making the virtual worlds even more interesting by setting them in non-Euclidean spaces.
     The objective of this thesis is to create a small video game incorporating both of the aforementioned concepts.
+    This work describes the implementation of a video game that incorporates both of the aforementioned concepts.
 
-    \noindent \textbf{Keywords:} keyword1, keyword2, ...
+    The game transforms Euclidean space into a hyperbolic or spherical space, which allows for the use of established algorithms.
+    This approach simplifies the implementation of the game logic in different geometries.
+    It also allows for changing the curvature of the space at runtime, which is the best way to showcase the differences between the geometries.
+    This approach also could theoretically be used to create a game that takes place in a world that is a combination of different geometries.
+
+    The game also features a number of classic game mechanics, such as terrain editing, character animations, day and night cycle, and many more.
+    This makes learning about the non-Euclidean geometry more fun and engaging.
```

Comment:
I'm uncertain about the educational merit of this stuff.
I'd stick to something like: "As a result, the game offers a unique experience of exploring and interacting with worlds inherently different from the one we live in."